components/list/test/list-nested.visual-diff.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../../../test/styles.css" type="text/css">
+		<script>
+			if (window.location.search.includes('dir=rtl')) {
+				document.documentElement.setAttribute('dir', 'rtl');
+			}
+		</script>
+		<script type="module">
+			import '../../typography/typography.js';
+			import './nested-list-iterations-helper.js';
+		</script>
+		<title>d2l-list-nested</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1.0">
+		<meta charset="UTF-8">
+	</head>
+	<body class="d2l-typography">
+
+		<div style="width: 700px;">
```

Comment:
Could the viewport have just been set to 700px wide? Or maybe the test element itself (`:host { width: 700px; }`)? Just hoping to avoid extra stuff in these HTML files for when we have to migrate them. :)

Reply:
So I added this here instead of using `viewport` to make the pages look better when you go to them directly outside of running the tests, which won't be a thing once we migrate 😅. I'll add it to the component for now at least